src/app/ui/header/header.component.html

```diff
@@ -20,19 +23,30 @@
             <li class="nav-item">
               <a class="nav-link" routerLink="/courses">Courses</a>
             </li>
-            <!--<li class="nav-item">-->
-              <!--<a class="nav-link" routerLink="/upload">Upload</a>-->
-            <!--</li>-->
             <!--<li class="nav-item">-->
               <!--<a class="nav-link disabled" href="#">ScoreBoard</a>-->
             <!--</li>-->
           </ul>
+
           <form class="form-inline my-2 my-lrg-0">
             <input class="form-control mr-sm-2" type="search" placeholder="Search course..." aria-label="Search">
-            <button class="btn btn-outline-success mr-2" type="submit" routerLink="/courses">Search</button>
-            <button class="btn btn-outline-primary" type="button" routerLink="/upload">Upload scan</button>
+            <button class="btn btn-outline-info mr-2" type="submit" routerLink="/courses">Search</button>
```

Comment:
If there is more than one button, all of them should be wrapped in a button group.
Wrap all buttons with a div with class `btn-group`.

Reply:
without `btn-group` it looks like that: 
![image](https://user-images.githubusercontent.com/19504086/50375784-b43fd580-060b-11e9-9549-b8a808d8e2e3.png)

with `btn-group` it looks like that: 
![image](https://user-images.githubusercontent.com/19504086/50375795-db96a280-060b-11e9-9dda-eaea5ed37660.png)

Personally, I prefer the first one.